docs/api.rst

```diff
@@ -392,6 +392,91 @@ Handling user creation, sign-in, log-out and querying
    :statuscode 500: Internal rotki error
    :statuscode 502: The external premium service could not be reached or returned unexpected response.
 
+Querying premium devices
+========================
+
+.. http:get:: /api/(version)/premium/devices
+
+   By doing a ``GET`` at this endpoint you can query the list of registered devices for your premium account.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      GET /api/1/premium/devices HTTP/1.1
+      Host: localhost:5042
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+          "result": {
+              "devices": [
+                  {
+                      "device_name": "MacOS",
+                      "user": "yabirgb",
+                      "device_identifier": "21312312"
+                  }
+              ],
+              "limit": 5
+          },
+          "message": ""
+      }
+
+   :resjson object result: Object containing devices list and limit information
+   :resjson list result.devices: List of registered devices for the premium account
+   :resjson string result.devices[].device_name: Name of the registered device
```

Comment:
I don't think this is how we show lists of objects in the rest of the docs. Does this render correctly if you build the docs?